packages/xkeyboard_config.rb

```diff
@@ -10,6 +10,8 @@ class Xkeyboard_config < Package
 
   depends_on "util_macros"
   depends_on "intltool"
```

Comment:
This is likely a build dependency.

Reply:
Is there any difference if  marked as "build"?

Reply:
The difference is this package is not installed needlessly if the binary build doesn't require it as a runtime dependency.  Since this is a small package, it doesn't really make much difference but when you include the go or jdk8 packages needlessly, this can take up quite a bit of disk space.  An example would be the hub package which only requires go to compile, not to run.